css/education.css

```diff
@@ -155,4 +155,149 @@ h5 {
       margin-top: 11rem;
       padding-left: 40px;
     }
-  }
```

Comment:
Please don't add or subtract lines. Such changes shouldn't be pushed. Its not a good practice.